features/fixtures/news_and_communications.json

```diff
@@ -31,6 +31,25 @@
     },
     "format_name":"News or communiqué",
     "show_summaries":true,
+    "sort": [
+      {
+        "name": "Most viewed",
+        "key": "-popularity"
+      },
+      {
+        "name": "Relevance",
+        "key": "-relevance"
+      },
+      {
+        "name": "Updated (newest)",
+        "key": "-public_timestamp",
+        "default": true
+      },
+      {
+        "name": "Updated (oldest)",
+        "key": "public_timestamp"
```

Comment:
It might be nice to have a list of sort options one can default to. If the above are going to be quite common it would save us republishing finder content items.